Codecademy/gamut
# fix: made social media links a grouped list

### Overview



Updating SocialMediaSharing group component to be an unordered list with SocialMediaLink as list items.



### PR Checklist

- [ ] Related to JIRA ticket: [DISC-1122](https://codecademy.atlassian.net/browse/DISC-1122)

#### Testing Instructions
_Gamut instructions_
1. Go to[ /?path=/docs/brand-labs-molecules-socialmediasharing--social-media-sharing](https://6435b2fb2d1c67368f07c34c--gamut-preview.netlify.app/?path=/docs/brand-labs-molecules-socialmediasharing--social-media-sharing)
2. Using VoiceOver
2. Use CONTROL + OPTION + RIGHT ARROW keys to navigate the Social media icons(Facebook, LinkedIn & twitter)
3. Voiceover should announce that the user is in a list
4. Inspect to ensure that the social media icons are individual `li` elements within `ul` element

_Monolith Instructions_
### learner story
1. Go to a learner story here: https://pr-35303-monolith.dev-eks.codecademy.com/learner-stories/web-development/5QNjwhzuCwfMfSPezJhWra
2. Social media sharing component should be 1-1 with what is in production

_Left is update, right is production_
<img width="742" alt="Screen Shot 2023-04-13 at 12 53 04 PM" src="https://user-images.githubusercontent.com/16326269/231844003-02c68f7f-be64-41b0-98f4-5125b5a09adf.png">

### sorting quiz
1. Log in
2. Go to [explore/sorting-quiz/results](https://pr-35303-monolith.dev-eks.codecademy.com/explore/sorting-quiz/results?advocator=38&connector=44&questioner=40&solver=37)
3. Social media sharing component should be 1-1 with what is in production

_Left is update, right is production_
<img width="872" alt="Screen Shot 2023-04-13 at 12 54 42 PM" src="https://user-images.githubusercontent.com/16326269/231844237-249594eb-3d14-4b52-9127-f8d12839e864.png">


### 30 day challenge confirmation modal
1. Log in
2. Go to [/30daychallenge/reminders](https://pr-35303-monolith.dev-eks.codecademy.com/30daychallenge/reminders)
3. Social media sharing component should be 1-1 with what is in production

_Left is update, right is production_
<img width="1121" alt="Screen Shot 2023-04-13 at 1 45 56 PM" src="https://user-images.githubusercontent.com/16326269/231844212-7f4c374c-f7e4-44a2-97d2-2cd299330c26.png">


#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/35303)  | [Monolith Env](https://pr-35303-monolith.dev-eks.codecademy.com/catalog?PR_ENV=pr-35303-portal) |



[DISC-1122]: https://codecademy.atlassian.net/browse/DISC-1122?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
